translation/source/activity.xml

```diff
@@ -72,4 +72,22 @@
     <item quantity="one">Joined %s team</item>
     <item quantity="other">Joined %s teams</item>
   </plurals>
+  <string name="chessInsights">Chess Insights</string>
```

Comment:
Not the best file to put the sources, activity refer to actions done by a player and can be seen on their profile, e.g https://lichess.org/@/Zhigalko_Sergei site.xml would be a better pick, or create an insight.xml